tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -19,42 +19,48 @@ local tpus = import 'templates/tpus.libsonnet';
 local utils = import 'templates/utils.libsonnet';
 
 {
-  local command_common = |||
-    pip install onnx
-    git clone --recursive https://github.com/pytorch-tpu/examples.git
-    python examples/deps/dlrm/dlrm_tpu_runner.py \
-      --arch-sparse-feature-size=64 \
-      --arch-mlp-bot=512-512-64 \
-      --arch-mlp-top=1024-1024-1024-1 \
-      --arch-interaction-op=dot \
-      --lr-num-warmup-steps 10 \
-      --lr-decay-start-step 10 \
-      --num-batches=1000 \
-      --data-generation="random" \
-      --numpy-rand-seed=727 \
-      --print-freq 100 \
-      --num-indices-per-lookup=100 \
-      --use-tpu \
-      --metrics-debug \
-      --num-indices-per-lookup-fixed \
-  |||,
-  local dlrm = common.PyTorchTest {
-    modelName: 'dlrm',
+  local dlrm = {
+    local config = self,
 
+    modelName: 'dlrm',
     volumeMap+: {
       datasets: common.datasetsVolume,
     },
-    podTemplate+:: {
-      spec+: {
-        containerMap+: {
-          train+: {
-            resources+: {
-              requests: {
-                cpu: '9.0',
-                memory: '30Gi',
-              },
-            },
-          },
+    paramsOverride:: {
+      scriptPath: 'tpu-examples/deps/dlrm/dlrm_tpu_runner.py',
+      miniBatchSize: 256,
+      archEmbeddingSize: '1000000-1000000',
+      tpuModelParallelGroupLen: 1,
+      tpuCores: 1,
+      ArchSparseFeatureSize: 64,
+      ArchMlpBot: '512-512-64',
+      ArchAlpTop: '1024-1024-1024-1',
+      NumIndicesPerLookup: 100,
+      trainCommand: [
+        'python3',
+        self.scriptPath,
+        '--arch-interaction-op=dot',
+        '--lr-num-warmup-steps=10',
+        '--lr-decay-start-step=10',
+        '--num-batches=1000',
+        '--data-generation=random',
+        '--numpy-rand-seed=72',
+        '--print-freq=100',
+        '--use-tpu',
+        '--metrics-debug',
+        '--num-indices-per-lookup-fixed',
+        '--mini-batch-size=%d' % config.paramsOverride.miniBatchSize,
+        '--arch-embedding-size=%s' % config.paramsOverride.archEmbeddingSize,
+        '--tpu-model-parallel-group-len=%d' % config.paramsOverride.tpuModelParallelGroupLen,
+        '--tpu-cores=%d' % config.paramsOverride.tpuCores,
+      ],
+    },
+    cpu: '9.0',
+    memory: '30Gi',
+    metricConfig+: {
```

Comment:
Do we need to disable TB assertions for this test? If not, please remove this `metricConfig` override

Reply:
right, btw what is TB assertions?

Reply:
TensorBoard assertions. i.e. test expectations based out TensorBoard metrics